0-ad-history-mod-package/simulation/templates/gaia/fauna_tiger.xml

```diff
@@ -23,6 +23,14 @@
     <GenericName>Tiger</GenericName>
     <SpecificName>Panthera tigris</SpecificName>
     <Icon>gaia/fauna_tiger.png</Icon>
+    <History>
+    “Dragonflies, katydids, but mostly chewed-up little kids”-- Hobbes on what Tigers are made out of: Calvin and Hobbes.
+    
+    
+    Tigers are the largest species of cat, with size reaching 9.5 feet long. In ancient times their range could have reached through Asia up to modern day Turkey. However in the modern day they are only found in small patches in southeast Asia.The tiger however could easily survive anywhere there is dense cover, water and large enough prey to sustain it.
+    Tigers hunt at night. Usually it takes deer and wild pigs, but thanks to it size can attack small rhinoceroses and elephants. It would not have any problem taking out cattle, or even an unprepared man.
```

Comment:
```suggestion
    Tigers hunt at night. Usually it takes deer and wild pigs, but thanks to its size can attack small rhinoceroses and elephants. It would not have any problem taking out cattle or even an unprepared man.
```